### DISCUSSION TITLE
Me apoyan

### DISCUSSION THREAD

**Discussion by Alanvalenci** [community/community#112605] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Quiero actualizar firefox pero siempre intervienen se los agradecería si me dejarán actualizarlo gracias en dado caso que tenga una duda o solicite su apoyo ya se los estaré compartiendo

**Comment by queenofcorgis** [comment]
Hi @Alanvalenci ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Firefox. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Alanvalenci** [comment]
Hola

Perdón 😥. Pero eso es lo que complementa gib hub un sistema con alta
seguridad para el código abierto ya que si no hay privacidad no hay
seguridad no volverá a suceder.
